.env-sample

```diff
@@ -1 +1,11 @@
 GRAPHQL_URL=
+
+FIREBASE_API_KEY=
+FIREBASE_AUTH_DOMAIN=
+FIREBASE_DATABASE_URL=
+FIREBASE_PROJECT_ID=
+FIREBASE_STORAGE_BUCKET=
+FIREBASE_MESSAGING_SENDER_ID=
+
+GOOGLE_OAUTH_IOS=
+GOOGLE_OAUTH_ANDROID=
```

Comment:
You are aware that all of these values will be available for anyone who downloads the app and reverse-engineers it

Reply:
Yes, these Firebase values should be public, it is even easier to obtain it on web.
Where something like this is suggested (which can look even scarier):
<img width="757" alt="Screenshot 2019-04-01 at 16 16 24" src="https://user-images.githubusercontent.com/2660330/55334657-f180f280-5499-11e9-919d-cb0b1f0f2451.png">

But you can/should then restrict login to certain domain/s (default is just localhost) in Firebase administration. And also set database rules to restrict what and by who can be written and read.

Reply:
Ok, it just sounds scary to provide apiKey to everyone, but if you are sure that it is not a problem...

Reply:
You can close down the access in the Google Console to say that the API key can only be used from certain origins, etc. so I think that's why it's okay, even though you should still take measures to restrict access to the key.

Reply:
Just one thing here: there is no newline at the end of the file and it can be problematic. I know that in the monorepo, they are careful with this.
It makes some common tools for counting lines in files, etc. not working as intended: https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Thanks, good to know. Extra line added.